src/Stock.js

```diff
@@ -100,14 +144,24 @@ class Stock extends React.Component {
                                 marker: { color: 'red' },
                             }
                         ]}
-                        layout={{ width: 800, height: 600, title: `${this.state.graphMode} ${this.state.stockTicker}` }}
-                    /> :
+                        layout={{ width: 800, height: 600, title: `${this.state.graphMode}${this.state.companyName}` }} /><div className="stockInfo">
```

Comment:
Can you put your stock info div on a new line (and maybe even with a line separating this component and the last)?

Reply:
Decided I would just do this change (as I already had it done while checking to see if it would help the flow of the code)